src/main/webapp/resources/css/mystyle.css

```diff
@@ -51,3 +51,64 @@
 	font-size: 12px;
 	border: 0 none;
 }
+.search-bar{
+    box-sizing: border-box;
+    position: relative;
+    table-layout: fixed;
+    width: 100%;
+    padding-left: 150px;
+    padding-right: 150px;
+    text-align: center;
+}
+.search-bar-row {
+    position: relative;
+    background-color: #ffffff;
+    border: 1px solid #bdbdbd;
+    box-shadow: 0 1px 3px 0 #bdbdbd;
+    border-radius: 4px;
+    margin: 0 auto;
+    display: table;
+    width: 100%;
+}
+.search-bar .search-bar-location,
+.search-bar .search-bar-radius,
```

Comment:
you can remove .search-bar (leave only .search-bar-radius etc), like:
.search-bar-location,
.search-bar-radius,
.search-bar-date,
.search-bar-submit {
    display: table-cell;
    text-align: center;
    vertical-align: middle;
    padding: 10px 16px 10px;
}